modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -48,19 +48,43 @@ dfx canister install -all
 +
 For a preview of how to use the simplified developer workflow, see https://sdk.dfinity.org/docs/quickstart/dfx-deploy-quickstart.html[Developer Workflow Preview].
 
-* You can now specify arguments on the command-line when you deploy a single canister using the `+dfx canister install+` command.
+* You can now specify arguments on the command-line when you deploy a **single** canister using an actor class with the `+dfx canister install+` command.
 +
-For example, you might use the following command to add the xx argument when deploying the canister named profile_map locally.
+For example, you might use the following command to add the `+self+` Text argument when deploying the canister named `+profile_map+` locally.
 +
 ....
-dfx canister install profile_map xx
+dfx canister install profile_map '("self")'
 ....
++
+Note that you specify the argument you want to pass using the Candid interface description format.
 
 * A new `+--memory allocation+` command-line option enables you to specify a memory allocation when deploying a canister.
 
 * The `+dfx identity new+` command has been enhanced to display a confirmation message when adding a new identify is successful.
 
-== Highlights of what's new in 0.6.7
+* Key updates for {proglang} include backend support for the following features:
+
+** Preliminary support for importing and exporting actor classes and for dynamic canister installation.
```

Comment:
```suggestion
** Preliminary support for importing actor classes.
** Preliminary support for dynamic canister installation (by calling the constructor of an imported actor class).
```

Reply:
Here's where it came from ...
== 0.4.4 (2020-09-21)
= Motoko compiler changelog

== 0.4.4 (2020-09-21)
Actor class export
Accept unit installation args for actors
Reject platform actor (class) programs with additional decs
Handle IO exceptions at the top-level
RTS: Remove duplicate array and blob allocation code
RTS: Fix pointer arithmetic in BigInt collection function

== 0.4.3 (2020-09-14)

Preliminary support for actor class import and dynamic canister installation. Surface syntax may change in future.
BREAKING CHANGE: a compilation unit/file defining an actor or actor class may only have leading import declarations; other leading declarations (e.g. let or type) are no longer supported.
Rust GC